plugins/modules/vios.py

```diff
@@ -478,6 +493,68 @@ def installVios(module, params):
 
     return changed, vios_property, warn_msg
 
+def installViosUsingDisk(module, params):
+    hmc_host = params['hmc_host']
+    hmc_user = params['hmc_auth']['username']
+    password = params['hmc_auth']['password']
+    vios_iso = params['vios_iso']
+    image_dir = params['image_dir']
+    vios_IP = params['vios_IP']
+    gateway_ip = params['gateway_ip']
```

Comment:
can you rename this params
gateway_ip  = vios_gateway
subnet_mask = vios_subnetmask